recipe/deploy/copy_dirs.php

```diff
@@ -7,7 +7,7 @@
         foreach (get('copy_dirs') as $dir) {
             if (test("[ -d {{previous_release}}/$dir ]")) {
                 run("mkdir -p {{release_path}}/$dir");
-                run("rsync -av {{previous_release}}/$dir/ {{release_path}}/$dir");
```

Comment:
Should we check what `$dir` does not contain `/` at the end? 

Will be nice to throw an exception with explanation of why to remove `/` from copy_dirs.

Reply:
I can also just trim `/` from the provided strings, in that case we don't need to throw an exception, what do you think?

Reply:
I think better to throw exception with explanation. In future it will help.

Reply:
Done.